Specs/Core/TaskProcessorSpec.js

```diff
@@ -3,29 +3,24 @@ import { TaskProcessor } from '../../Source/Cesium.js';
 import absolutize from '../absolutize.js';
 import { when } from '../../Source/Cesium.js';
 
-xdescribe('Core/TaskProcessor', function() {
+describe('Core/TaskProcessor', function() {
 
     var taskProcessor;
 
     beforeEach(function() {
-        TaskProcessor._workerModulePrefix = '../Specs/TestWorkers/';
-
-        TaskProcessor._loaderConfig = {
-            baseUrl : absolutize(require.toUrl('Source'))
-        };
+        TaskProcessor._workerModulePrefix = absolutize('../Specs/TestWorkers/');
     });
 
     afterEach(function() {
         TaskProcessor._workerModulePrefix = TaskProcessor._defaultWorkerModulePrefix;
-        TaskProcessor._loaderConfig = undefined;
 
         if (taskProcessor && !taskProcessor.isDestroyed()) {
             taskProcessor = taskProcessor.destroy();
         }
     });
 
     it('works with a simple worker', function() {
-        taskProcessor = new TaskProcessor('returnParameters');
+        taskProcessor = new TaskProcessor('returnParameters.js');
```

Comment:
Why do these need the `.js`? The actual source code calls don't seem to need this?